app/serializers/provider_serializer.rb

```diff
@@ -4,7 +4,12 @@ class ProviderSerializer < ActiveModel::Serializer
   type 'cc_provider'
 
   def id
-    "ccp_#{object.ProviderIdentifier}"
+    if Flipper.enabled?(:facility_locator_ppms_forced_unique_id)
+      obj_id = object.ProviderHexdigest || object.ProviderIdentifier
+      "ccp_#{obj_id}"
+    else
+      "ccp_#{object.ProviderIdentifier}"
+    end
```

Comment:
A minor re-factoring suggestion, but non-blocking:

```suggestion
obj_id = object.ProviderHexdigest if Flipper.enabled?(:facility_locator_ppms_forced_unique_id)
obj_id ||= object.ProviderIdentifier
"ccp_#{obj_id}"
    end
```

Reply:
so you really don't like `obj_id = object.ProviderHexdigest || object.ProviderIdentifier`

Reply:
personally, I think the `if ... else` structure is clearer and I'm fine with the `||`.  But I think as a matter of style both are acceptable